submit_cgap/submission.py

```diff
@@ -514,7 +515,15 @@ def resume_uploads(uuid, server=None, env=None, bundle_filename=None, keydict=No
                        subfolders=subfolders)
 
 
-def execute_prearranged_upload(path, upload_credentials):
+def get_s3_encrypt_key_id(auth):
+    try:
+        health = get_health_page(key=auth)
+        return health.get('s3_encrypt_key_id')
```

Comment:
This will return a value in the current state on non-orchestrated envs - that may be a problem here? https://cgap.hms.harvard.edu/health

Reply:
I think it's not a problem because the only use of this tests whether this is false before adding it to arguments to use.